--- hurwitz.lyx	2007-02-10 22:29:05.000000000 -0600
+++ hurwitz-arxiv.lyx	2007-02-10 22:29:27.000000000 -0600
@@ -3792,9 +3795,8 @@ Periodic zeta function
 
 
 \begin_inset Graphics
-	filename periodic-xsection.png
+	filename periodic-xsection.ps
 	width 100text%
-	special bb=0 0 640 480
 
 \end_inset 
 
@@ -4262,9 +4264,8 @@ The Periodic Zeta
 
 
 \begin_inset Graphics
-	filename periodic-50-hires.png
+	filename periodic-50.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -4404,9 +4405,8 @@ Hurwitz zeta, with leading term subtract
 
 
 \begin_inset Graphics
-	filename hurwitz-non-leading.png
+	filename hurwitz-non-leading.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -4478,9 +4478,8 @@ Extended view of Hurwitz zeta
 
 
 \begin_inset Graphics
-	filename hurwitz-extended-hires.png
+	filename hurwitz-extended.ps
 	width 100text%
-	special bb= 0 0  600 600
 
 \end_inset 
 
@@ -4570,9 +4569,8 @@ Polylogarithm phase plot
 
 
 \begin_inset Graphics
-	filename poly-phase.png
+	filename poly-phase.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -4694,9 +4692,8 @@ Polylog whole plane
 
 
 \begin_inset Graphics
-	filename polylog-0-sheet.png
+	filename polylog-0-sheet.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -4797,9 +4794,8 @@ Polylogarithm off the critical axis
 
 
 \begin_inset Graphics
-	filename polylog-1.2-14.png
+	filename polylog-1.2-14.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -4895,9 +4891,8 @@ Upper sheet of the polylogarithm
 
 
 \begin_inset Graphics
-	filename polylog-p1-sheet.png
+	filename polylog-p1-sheet.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -5009,9 +5004,8 @@ Lower sheet of the polylogarithm
 
 
 \begin_inset Graphics
-	filename polylog-m1-sheet.png
+	filename polylog-m1-sheet.ps
 	width 100text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -5086,28 +5080,25 @@ More polylogarithm sheets
 
 
 \begin_inset Graphics
-	filename polylog-m1-sheet.png
+	filename polylog-m1-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
 
 \begin_inset Graphics
-	filename polylog,-0,-2-sheet.png
+	filename polylog,-0,-2-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
 
 \begin_inset Graphics
-	filename polylog,-0,-3-sheet.png
+	filename polylog,-0,-3-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
@@ -5116,28 +5107,25 @@ More polylogarithm sheets
 
 
 \begin_inset Graphics
-	filename polylog,1,1-sheet.png
+	filename polylog,1,1-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
 
 \begin_inset Graphics
-	filename polylog,1,-2-sheet.png
+	filename polylog,1,1-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
 
 \begin_inset Graphics
-	filename polylog,1,-3-sheet.png
+	filename polylog,1,1-sheet.ps
 	lyxscale 25
 	width 33text%
-	special bb=0 0 600 600
 
 \end_inset 
 
